lib/ui/list_item.dart

```diff
@@ -9,34 +11,76 @@ class ResultListItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final textStyle = TextStyle(fontWeight: FontWeight.normal, fontSize: TextSize.mediumTitle);
+    final pointsStyle = TextStyle(
+      fontWeight: FontWeight.w700,
+      fontFamily: FontFamily.roboto,
+      fontSize: TextSize.mediumTitle,
+      color: Colors.white,
+    );
+
+    final pktStyle = TextStyle(
```

Comment:
pointsStyle i pktStyle brzmią jak jedno i to samo. Proponuje nazwy "pointValueStyle", "pointDescriptionStyle".